### DISCUSSION TITLE
generating certs for domains and sites

### DISCUSSION THREAD

**Discussion by ghost** [chris2511/xca#604] [Q&A]
what certificates do i need so i can issue certificates for my sites in XCA

**Comment by itsKV** [comment]
You will need following 'KEY USES'
_Digital Signature 
Key Encipherment_

Also, following extended/enhanced key uses are required 
_Server Authentication 
Client Authentication_

Refer [this](https://knowledge.digicert.com/solution/what-extensions-and-details-are-included-in-a-ssl-certificate) page for further certificate contents details.
